IIPS/iip-52.md

```diff
@@ -169,73 +169,70 @@ At this time, the `_from` would be the BTP address of `xcall`.
 
 ### Fees Handling
 
-The RI for ICON network has adopted a simple fixed fees handling mechanism. If a user wants to make a call from
-ICON to the T1 network, he needs to pay X ICX, and for the T2 network he needs to pay Y ICX.  That is, the fees
-depend on the destination network address.  Also, a `default` fee has been defined in case of there is no fee
-mapping to the network address.
+If a user wants to make a call from ICON to the T1 network, he needs to pay X ICX, and for the T2 network
+he needs to pay Y ICX.  That is, the fees depend on the destination network address.
```

Comment:
Any other insight into how the fee is determined? Additionally, why is the `Fee Handling` section being discussed in the IIP for `ICON BTP Arbitrary Call Service Standard`? Are these fees specific to the xcall standard, or are they a more general topic?

Reply:
As BTP is an open framework, there are no rules that enforce that someone else who implements BTP could not define a separate service. For example, a different arbitrary call service (e.g. arbCall) with different rules, but with the same fee structure

These two pieces of information are sufficiently separate that I believe it should be a separate IIP. Something like the following:

`IIP: BTP Service Fee Handling`